decidim-accountability/lib/decidim/api/result_type.rb

```diff
@@ -17,12 +22,23 @@ class ResultType < Decidim::Api::Types::BaseObject
       field :id, GraphQL::Types::ID, "The internal ID for this result", null: false
       field :parent, Decidim::Accountability::ResultType, "The parent result", null: true
       field :progress, GraphQL::Types::Float, "The progress for this result", null: true
-      field :reference, GraphQL::Types::String, "The reference for this result", null: true
+      field :proposal_urls, [String, { null: true }], "The proposal URLs for this result", null: true
```

Comment:
On this case we could just return the `ProposalType`, right? I think it'd be more idiomatic, as if we just return the URL, then the consumers of the API need to make another query for getting the metadata of the Proposals by getting the ID/component/process/etc.  

Does that makes sense?